docs/advanced_reference/commands/autonomy_service.md

```diff
@@ -38,8 +38,56 @@ This command group consists of a number of functionalities to manage the life cy
 
     The options `--use-ethereum`, `--use-goerli`, `--use-custom-chain` and `--use-local` are mutually exclusive.
 
-`--skip-hash-check`
-: Skip hash check when verifying dependencies on chain.
+`-t, --timeout FLOAT`
+: Timeout for verifying emitted events
```

Comment:
@jmoreira-valory `autonomy service` does not have `--skip-hash-check` flag and the doc was missing the `--timeout` flag for the same